simp_le.py

```diff
@@ -1469,11 +1469,17 @@ def persist_new_data(args, existing_data):
                     'This client is unable to solve any other challenges.')
 
     for name, auth in six.iteritems(authorizations):
-        challb = supported_challb(auth)
-        response, validation = challb.response_and_validation(client.net.key)
-        save_validation(roots[name], challb, validation)
-
-        client.answer_challenge(challb, response)
+        for _ in range(5):
+            time.sleep(1)
```

Comment:
Let's put the call to sleep() at the end of the loop, so if the first attempt is successful we don't wait.